leits/MeetingBar
# feat(statusbar): detect if meetingbar is not displayed

### Status
**READY**

### Description
- we now detect if the meetingbar icon is not displayed in the status bar
- we will now send a notification if meetingbar is not dispayed in the status bar due to low space
- the user can deactivate the notification in the preferences dialog or directly in the notification

![image](https://user-images.githubusercontent.com/3872101/161443170-c9d98ddf-2ffb-4938-80aa-9923d2924cd8.png)

![image](https://user-images.githubusercontent.com/3872101/161443178-14584ba4-a040-4340-abce-5aa823e9ab50.png)


## Checklist

### Steps to Test or Reproduce

- open many apps with statusbar icons 
- open meetingbar at least, so it is not displayed (try to configure the displayed length of the title in status bar to the maximum)
- notice the notification
